pytrip/tripexecuter/plan.py

```diff
@@ -202,9 +184,6 @@ def _print(self):
         out += "|\n"
         out += "| Directories\n"
         out += "|   Working directory           : {:s}\n".format(self.working_dir)
-        out += "|   DDD directory               : {:s}\n".format(self.ddd_dir)
```

Comment:
User should have the option to see the DDD/SPC/SIS paths when plan/field/kernel objects are printed. Where is now the place where DDD/SPC/SIS paths are printed ?

Reply:
DDD/SPC/SIS are kernel specific, so they are handle by KernelModule
https://github.com/pytrip/pytrip/blob/2c73bea76fd59a08e1ac367d1a5f652c08b3d9ea/pytrip/tripexecuter/kernel.py#L58

Reply:
ok